### DISCUSSION TITLE
help!

### DISCUSSION THREAD

**Discussion by coolman123jj** [dnschneid/crouton#4545] [Q&A]
Im quite confused by how to download it, can anyone help me?

**Comment by DennisLfromGA** [comment]
Download the crouton installer with this link: https://goo.gl/fd3zc

For usage see: https://github.com/dnschneid/crouton#usage

**Comment by ghost** [comment]
I've written a [guide](https://mrhallwayblog.wordpress.com/2020/09/18/installing-linux-on-a-chromebook/) on how to use it. Although, it is suggested to use the [README](https://github.com/dnschneid/crouton/blob/master/README.md) file.
